ray-operator/test/e2e/rayjob_lightweight_test.go

```diff
@@ -5,17 +5,19 @@ import (
 
 	. "github.com/onsi/gomega"
 
-	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	corev1ac "k8s.io/client-go/applyconfigurations/core/v1"
 
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
why split k8serrors import away from the other k8s.io/apimachinery imports?

Reply:
Ah actually I didn't notice that. It's changed by the linter.